sample-code/spring-app/src/main/java/com/sap/ai/sdk/app/services/SpringAiOrchestrationService.java

```diff
@@ -263,4 +265,17 @@ public Translation responseFormat() {
             "How do I say 'AI is going to revolutionize the world' in dutch?", defaultOptions);
     return cl.prompt(prompt).call().entity(Translation.class);
   }
+
+  /**
+   * Create an embedding for a given text using the Orchestration service.
+   *
+   * @param inputText the text to embed
+   * @return the embedding as a float array
+   */
+  @Nonnull
+  public float[] embed(@Nonnull final String inputText) {
+    val embedOptions =
+        EmbeddingOptionsBuilder.builder().withModel("text-embedding-3-small").build();
```

Comment:
```suggestion
        EmbeddingOptionsBuilder.builder()
            .withModel(OrchestrationEmbeddingModel.TEXT_EMBEDDING_3_SMALL.name())
            .build();
```
Optional, how do you feel about this usage?